main/src/modules/abm/abmReasoning/src/abmReasoning.cpp

```diff
@@ -186,6 +186,8 @@ bool abmReasoning::respond(const yarp::os::Bottle& bCommand, yarp::os::Bottle& b
 {
 
     bReply.clear();
+    bReply.addVocab(Vocab::encode("many"));
```

Comment:
Hi @gregoire-pointeau,
What is this used for? Was this added by mistake? Does it break compatibility with other modules which interact with `abmReasoning`?